papers/Raku_Steering_Council_Code.md

```diff
@@ -291,6 +295,136 @@ initial election, will be assumed to be inactive, and have their status
 changed to "inactive", thus revoking their voting rights until they've
 become active again and accepted by a core team vote.
 
+Raku users' community
+---------------------
+
+Anyone who has signed up to the raku users email list is considered a member of the Raku
```

Comment:
This is going to be hard, because a. we don't have a raku users mailing list, and b. many people do not use email anymore and/or do not want to be on a mailing list.

Reply:
> Thank you for your PR. Since the whole issue around Standards of Conduct / Code of Conduct have been extensively discussed in [Raku/problem-solving#134](https://github.com/Raku/problem-solving/issues/134) , with a stalled PR at [Raku/problem-solving#136](https://github.com/Raku/problem-solving/pull/136) , I think the RSC should make it one of the first points of business to resolve that. Adding a CoC now just like that, will not be acceptable to a lot of people.
> 
> In light of the handling of CoC's in different conferences in the past year, this feels **extra** sensitive at the moment: having a certain CoC has been reason for people to **not** visit a conference, and the absence of a CoC has _also_ been reason for people to **not** visit a conference. So this subject needs to be dealt with with the utmost of caution: just putting it in a PR will not work :-(
> 
> I will have a closer look at the other changes tomorrow.